### PR TITLE
Update test_model.py

### PR DESCRIPTION
**Pull request by AshkanTaghipour**
The unit test for the evaluation of Encoder, Decoder, and 2D attention module added.

### CODE REVIEW

File: testing/test_model.py [hunk 1]
```diff
@@ -52,5 +52,35 @@ def test_film_generator():
     assert (type(inf[0]) == torch.Tensor)
     assert (type(inf[1]) == torch.nn.parameter.Parameter)
 
+def test_modified_Encoder():
```
Comment (jcohenadad) [inline testing/test_model.py:55]: functions are in lower case according to [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Comment (AshkanTaghipour) [reply testing/test_model.py:55]: You're right, I will follow and revise it into the correct form.

**Comment by jcohenadad** [comment]
@AshkanTaghipour could you please use explicit commit messages (unlike [here](https://github.com/ivadomed/ivadomed/pull/555/commits)), so it is easier to follow the git history of your commits? thanks

**Comment by AshkanTaghipour** [comment]
> @AshkanTaghipour could you please use explicit commit messages (unlike [here](https://github.com/ivadomed/ivadomed/pull/555/commits)), so it is easier to follow the git history of your commits? thanks

Yes of course.

**Comment by jcohenadad** [comment]
> > @AshkanTaghipour could you please use explicit commit messages (unlike [here](https://github.com/ivadomed/ivadomed/pull/555/commits)), so it is easier to follow the git history of your commits? thanks
> 
> Yes of course.

i see that your latest commit is more verbose https://github.com/ivadomed/ivadomed/pull/555/commits/1d1881b9555c11c3d962262942e2732d3c10b935, however please also use a more explicit commit message-- see: https://www.neuro.polymtl.ca/software/contributing#committing

**Comment by AshkanTaghipour** [comment]
> > > @AshkanTaghipour could you please use explicit commit messages (unlike [here](https://github.com/ivadomed/ivadomed/pull/555/commits)), so it is easier to follow the git history of your commits? thanks
> > 
> > 
> > Yes of course.
> 
> i see that your latest commit is more verbose [1d1881b](https://github.com/ivadomed/ivadomed/commit/1d1881b9555c11c3d962262942e2732d3c10b935), however please also use a more explicit commit message-- see: https://www.neuro.polymtl.ca/software/contributing#committing

Thanks

**Comment by AshkanTaghipour** [comment]
dear @jcohenadad I had an assertion error in all encoder, decoder, and unet_att2d classes in the first commit, in the last commit, the error of the encoder class removed.
The remained error declared the unexpected keyword of "attention" which I think the testing is done on the Ivadomed models.py, not on the revised version which I added before, 
should I change the current version of model.py  by my revised version in the master?

**Comment by jcohenadad** [comment]
> dear @jcohenadad I had an assertion error in all encoder, decoder, and unet_att2d classes in the first commit, in the last commit, the error of the encoder class removed.
> The remained error declared the unexpected keyword of "attention" which I think the testing is done on the Ivadomed models.py, not on the revised version which I added before,
> should I change the current version of model.py by my revised version in the master?

the unit test should be added directly in #552 — that’s the point of testing (ie make sure it passes before merging new code in master)

**Comment by jcohenadad** [comment]
@AshkanTaghipour what is the status of this PR?

**Comment by andreanne-lemay** [comment]
The changes related to the attention U-Net will be integrated in #552. As discussed internally during the meeting, this PR is not relevant and can be closed.

**Pull request closed without merging**